spock-core/src/main/java/org/spockframework/compiler/WhereBlockRewriter.java

```diff
@@ -16,21 +16,27 @@
 
 package org.spockframework.compiler;
 
-import java.util.*;
```

Comment:
please don't change the ordering of the imports and also avoid whitespace changes, where nothing really happens, this makes this commit just harder to read.